bin/package-update.sh

```diff
@@ -39,3 +39,10 @@ find ./packages/woocommerce-admin -iname '*.js' -exec sed -i.bak -e "s/, 'woocom
 # Cleanup backup files
 find ./packages -name "*.bak" -type f -delete
 output 2 "Done!"
+
+# Apply patches
+output 2 "Applying patch #450 to Action Schduler"
+cd packages/action-scheduler
+curl -O https://patch-diff.githubusercontent.com/raw/woocommerce/action-scheduler/pull/450.patch
+patch -p1 < 450.patch
```

Comment:
[not a blocker] should it be ` patch -p1 < 450.patch && output 2 "Done!" `?

Reply:
Yeah... maybe not. Due to our dependency, I _think_ we need to run the following steps to get the package:

```
npm install
# this includes wp cli i18n package 
composer install
# this uses wp cli i18n package 
npm run build 
# to get rid of all packages we don't need
composer install --no-dev
```

Second patch fails (already applied) and I don't think we need to fix it now, as we should get rid of extra patching anyway.